src/shared/utils/browser/set-auth-cookie.ts

```diff
@@ -0,0 +1,15 @@
+import { isHttps } from "@utils/env";
+import * as cookie from "cookie";
+import { authCookieName } from "../../config";
+
+export default function setAuthCookie(jwt: string) {
+  const expires = new Date();
+  expires.setDate(expires.getDate() + 365);
+
+  document.cookie = cookie.serialize(authCookieName, jwt, {
+    expires,
```

Comment:
This still uses `expires`. Judging by the article I linked earlier, `expires` has more backward compatibility than `maxAge` (although it seems only some versions of internet explorer don't support `maxAge` - not sure if it's worth bothering). Either way, if `clearAuthCokkie` is using `maxAge`, this should as well. `maxAge` is in milliseconds.

Reply:
Good point. Is there any way you could quickly push a commit to fix that here? I'm about to be out of office for the rest of the day.